matchms/importing/load_from_msp.py

```diff
@@ -84,62 +135,18 @@ def get_peak_comment(rline: str) -> str:
 def parse_metadata(rline: str, params: dict):
     """ Reads metadata contained in line into params dict. """
     splitted_line = rline.split(":", 1)
-    if splitted_line[0].lower() == 'comments':
+    if splitted_line[0].lower() == 'comments' and "=" in splitted_line[1]:
```

Comment:
Why does the comment line have to contain an `=` sign?

Reply:
@florian-huber if we resolve the remaining points here I think we could merge the development branch.

Reply:
The `=` was added because I saw two types of comment field uses. In one there is a lot of pairs like `"license=CC BY-NC-SA" "exact mass=247.06333"` which are then addded to the metadata under the respective names. But it is also possible to use the comment field for any other arbitrary message I guess. In that case the former code would simply discard the comment field altogether. Now it will remain in the metadata as `comments`.

Reply:
Ah okay, I get it. Thanks for explaining!